pkgs/applications/misc/khal/default.nix

```diff
@@ -26,11 +26,16 @@ with python3.pkgs; buildPythonApplication rec {
     pkginfo
     freezegun
   ];
-  nativeBuildInputs = [ setuptools_scm ];
+  nativeBuildInputs = [ setuptools_scm sphinx sphinxcontrib_newsfeed ];
   checkInputs = [ pytest ];
 
   postInstall = ''
+    # zsh completion
     install -D misc/__khal $out/share/zsh/site-functions/__khal
+
+    # man page
+    make -C doc man
```

Comment:
Makefile doesn't support install for this?

Reply:
doesn't seem like it https://github.com/pimutils/khal/blob/master/doc/Makefile